x/leverage/types/errors.go

```diff
@@ -9,7 +9,7 @@ import (
 var (
 	ErrInvalidAsset            = sdkerrors.Register(ModuleName, 1100, "invalid asset")
 	ErrInsufficientBalance     = sdkerrors.Register(ModuleName, 1101, "insufficient balance")
-	ErrBorrowLimitLow          = sdkerrors.Register(ModuleName, 1102, "borrow limit too low")
+	ErrBorrowLimitLow          = sdkerrors.Register(ModuleName, 1102, "borrowing over the limit")
```

Comment:
Previous wording exists because `ErrBorrowLimitLow` applies to three scenarios:
- Borrowing too much
- Withdrawing too much collateral
- Disabling necessary collateral

The new wording should be suitable for all three cases

Reply:
OK, will update. But the original wording also doesn't clearly apply to the 3 cases you listed.

Reply:
Yeah, I couldn't find a good one either - let me know if you can (or we could also split it into multiple error types to allow for more specific wordings)

Reply:
I will create more error types.